js/fetch_types.ts

```diff
@@ -0,0 +1,426 @@
+type HeadersInit = Headers | string[][] | Record<string, string>;
```

Comment:
Because this contains no functional code, it doesn't need to be transpiled, and can just be renamed `fetch_types.d.ts` and be inlined in `assets.ts` just like the `js/types.d.ts` is now.  I think when we get rid of `lib.globals.d.ts` and change so that `globals.ts` is easier to maintain, it will impact how this file is handled as well, but we can cross the bridge when we get there.

Reply:
Good point - renamed to fetch_types.d.ts